docs/source/markdown/podman-container-prune.1.md

```diff
@@ -63,15 +65,15 @@ fff1c5b6c3631746055ec40598ce8ecaa4b82aef122f9e3a85b03b55c0d06c23
 
 Remove all stopped containers from local storage created within last 10 minutes
 ```
-$ sudo podman container prune --filter until="10m"
+$ podman container prune --filter until="10m"
 WARNING! This will remove all stopped containers.
 Are you sure you want to continue? [y/N] y
 3d366295e33d8cc612c4d873199bacadd55088d90d17dcafaa9a2d317ad50b4e
 ```
 
 ## SEE ALSO
-podman(1), podman-ps
+**[podman(1)](podman.1.md)**, **[podman-ps](podman-ps.1.md)**
 
 ## HISTORY
-December 2018, Originally compiled by Brent Baude (bbaude at redhat dot com)
-December 2020, converted filter information from docs.docker.com documentation by Dan Walsh (dwalsh at redhat dot com)
+December 2018, Originally compiled by Brent Baude <bbaude@redhat.com>\
+December 2020, converted filter information from docs.docker.com documentation by Dan Walsh <dwalsh@redhat.com>
```

Comment:
Did you want to leave a blank line after this?

Reply:
See [here](https://github.com/containers/podman/pull/10676#discussion_r656576475).